pkg/controller/certificates/issuing/internal/keystore.go

```diff
@@ -74,13 +75,11 @@ func encodePKCS12Keystore(profile cmapi.PKCS12Profile, password string, rawKey [
 }
 
 func encodePKCS12Truststore(profile cmapi.PKCS12Profile, password string, caPem []byte) ([]byte, error) {
-	ca, err := pki.DecodeX509CertificateBytes(caPem)
+	cas, err := pki.DecodeX509CertificateChainBytes(caPem)
```

Comment:
This function performs the correct operation, but the name is misleading.
In this case, we are decoding a pool of certificates, which do not form a chain.
Could you create a new pki function "DecodeX509CertificateSetBytes" and replace the implementation of "DecodeX509CertificateChainBytes" with a call to this new function?
This way, both functions still perform the same task, but there will be no confusion what the function is meant to be doing based on the name.

Reply:
That makes sense. I've updated the branch with the new utility alias.